packages/sandboxes/hello-world/index.tsx

```diff
@@ -0,0 +1,35 @@
+import * as LLMx from "ai-jsx";
+import {
+  ChatCompletion,
+  SystemMessage,
+  UserMessage,
+} from "ai-jsx/core/completion";
+import { pinoLogger } from "ai-jsx/core/log";
+import { pino } from "pino";
+
+function App() {
+  return (
+    <ChatCompletion>
+      <SystemMessage>
+        You are an agent that only asks rhetorical questions.
+      </SystemMessage>
+      <UserMessage>How can I learn about Ancient Egypt?</UserMessage>
+    </ChatCompletion>
+  );
+}
+const pinoStdoutLogger = pino({
+  name: "ai-jsx",
+  level: process.env.loglevel ?? "trace",
```

Comment:
```suggestion
  level: process.env.loglevel ?? "debug",
```

`trace` is pretty noisy – we might want to stick to `debug`. But I don't feel strongly.